torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -627,15 +663,18 @@ def _record_matching_score(
         ],
         attributes: Dict[str, fx_type_utils.Argument],
     ):
-        """Calculate the inputs matching score of the OpSchema requirements to find the nearest match.
+        r"""Calculate the inputs matching score of the OpSchema requirements to find the nearest match.
+
+        Only the functions which have the same number of inputs and attributes as the
+        OpSchema are eligible to be a nearest match candidate. Thus, we don't need to
```

Comment:
maybe check and raise otherwise in the beginning of this function

Reply:
It's guaranteed to have the same length of inputs and attributes because `self._record_matching_score(function_inputs, function_attributes)` is only invoked when the check are done in `perfect_match` function (step 1).